content/_guides/faq.md

```diff
@@ -197,8 +197,8 @@ in other places.
 If the attempts are very frequent and persistent, feel welcome to let staff
 know, just in case.
 
-_At your own risk_, and depending on your client, you may be able to filter
-the alerts. Ask your client's support channel if this is possible.
+_At your own risk_, you can mute these alerts.
+See `/msg Nickserv HELP SET BADPASSWDMSG`.
```

Comment:
We tend to give the requisite information directly in the guide/FAQ rather than requiring the user look it up via an IRC command. So, this section should just directly give the command to enable it (in a code block to match the formatting for other command examples on the page). For example, the FAQ entry at https://libera.chat/guides/faq#why-is-my-nickname-in-use directly states how to use RELEASE and REGAIN rather than directing the user to help text that isn't available on the website.

Reply:
Addressed.